pkg/network/tracer/tracer_linux_test.go

```diff
@@ -1715,3 +1718,98 @@ func TestBlockingReadCounts(t *testing.T) {
 
 	assert.Equal(t, uint64(n), conn.MonotonicSum().RecvBytes)
 }
+
+// GoTLS test
+
+// Test that we can capture HTTPS traffic from Go processes started after the
+// tracer.
+func TestHTTPGoTLSCaptureNewProcess(t *testing.T) {
+	const (
+		ServerAddr          = "localhost:8081"
+		ExpectedOccurrences = 10
+	)
+
+	if runtime.GOARCH != "amd64" {
+		t.Skip("GoTLS support not available on non amd64 architectures")
+	}
+
+	if !httpSupported(t) {
+		t.Skip("HTTPS feature not available on pre 4.14.0 kernels")
+	}
+
+	if !httpsSupported(t) {
+		t.Skip("HTTPS feature not available supported for this setup")
+	}
+
+	// Setup
+	closeServer := httptest.HTTPServer(t, ServerAddr, httptest.Options{
+		EnableTLS: true,
+	})
+	defer closeServer()
+
+	cfg := config.New()
+	cfg.EnableRuntimeCompiler = true
```

Comment:
why?

Reply:
We can only use GoTLS reliably with runtime compilation enabled. On prebuilt, the lack of offset guessing will make the test fail in the CI.

Reply:
For future reference: we agreed on not having offset guessing in this PR, but we will try to add it in the future, if possible.